test/modules/standard/gmp/ferguson/gmp_dist_array.chpl

```diff
@@ -44,16 +41,27 @@ modulus.nextprime(modulus);
 writeln(modulus);
 sum.mod(sum, modulus);
 writeln(sum);
-delete modulus;
-delete sum;
+
 
 // Release the array elements.
+// TODO: remove this section once the records are being released correctly.
```

Comment:
I'm not adding manual_free to the BigInt type module, just to that one test so that it leaks less (it still leaks 2000 bytes of other data related to leaking the array elements). 

If we would prefer to let the test leak everything, we could do that.